src/containers/ExhibitionDetail/index.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import axios from 'axios';
+
+import styles from './styles.module.css';
+
+const axiosInstance = axios.create({ baseURL: 'https://api.artic.edu/api/v1' });
```

Comment:
You've already defined an axios instance in the file `src/containers/Exhibitions/index.jsx`, which can be abstracted away in its file so that it can be reused across multiple container components such as this one and `Exhibitions`. Then you'd import the same instance from both files and thus removing the need to create a separate axios instance from each.